docs/DeveloperGuide.adoc

```diff
@@ -235,12 +315,91 @@ image::CommitActivityDiagram.png[]
 ** Cons: Requires dealing with commands that have already been undone: We must remember to skip these commands. Violates Single Responsibility Principle and Separation of Concerns as `HistoryManager` now needs to do two different things.
 // end::undoredo[]
 
-// tag::dataencryption[]
-=== [Proposed] Data Encryption
+=== Notification Feature
+A notification is a message that Mortago will display when the status of a body is automatically updated.
+If the next-of-kin of a body has not been uncontactable for 24 hours, Mortago will prompt the user to contact
+the police for a more thorough investigation and change the status of the body from `ARRIVED` to `CONTACT_POLICE`.
 
-_{Explain here how the data encryption feature will be implemented}_
+==== Current Implementation
+Given below is the execution process of the `NotificationCommand`:
 
-// end::dataencryption[]
+1. The user executes `add -b /name John ...` to add a new body to Mortago.
+2. The add command also created a new new `Notif` using this body as the parameter.
+3. The add command instantiates a new`NotifCommand` with `Notif`, `period`, `unit` as parameters.
+`period` is a static final integer initialized for 10 seconds and `unit` is static final TimeUnit set at TimeUnit.SECONDS.
+4. `NotifCommand` creates a new static `ScheduledExecutorService` which is scheduled to update the `BodyStatus` to
+`CONTACT_POLICE` if there is no change in its status for the specified `period` using the `UpdateCommand`.
+5. After the `BodyStatus` is updated, a `NotifWindow` will be shown to alert the user of the automatic update in
+`BodyStatus`.
+
+.Notification Command Sequence Diagram
+image::NotifCommandSequenceDiagram.png[]
```

Comment:
What's "up"?

Perhaps, more clearer variable names could be used.


Additionally, the source of the commands could be specified, instead of an arrow coming out of nowhere.

Reply:
The cross should not overlap with the box. The arrows should not be pointing to the end of the box but part of the box.